policy_packs/aws/backup/enforce_ebs_volume_backups/README.md

```diff
@@ -0,0 +1,183 @@
+
+## Usage
+
+### Install Policy Pack
+
+> [!NOTE]
+> By default, installed policy packs are not attached to any resources.
+>
+> Policy packs must be attached to resources in order for their policy settings to take effect.
+
+Clone:
+
+```sh
+git clone https://github.com/turbot/guardrails-samples.git
+cd guardrails-samples/policy_packs/aws/backup/enforce_ebs_volume_backups
+```
+
+Run the Terraform to create the policy pack in your workspace:
+
+```sh
+terraform init
+terraform plan
+```
+
+Then apply the changes:
+
+```sh
+terraform apply
+```
+
+### Apply Policy Pack
+
+Log into your Guardrails workspace and [attach the policy pack to a resource](https://turbot.com/guardrails/docs/guides/policy-packs#attach-a-policy-pack-to-a-resource).
+
+If this policy pack is attached to a Guardrails folder, its policies will be applied to all accounts and resources in that folder. The policy pack can also be attached to multiple resources.
+
+For more information, please see [Policy Packs](https://turbot.com/guardrails/docs/concepts/policy-packs).
+
+### Enable Enforcement
+
+> [!TIP]
+> You can also update the policy settings in this policy pack directly in the Guardrails console.
+>
+> Please note your Terraform state file will then become out of sync and the policy settings should then only be managed in the console.
+
+By default, the policies are set to `Check` in the pack's policy settings. To enable automated enforcements, you can switch these policies settings by adding a comment to the `Check` setting and removing the comment from one of the listed enforcement options:
+
+```hcl
+# AWS > Backup > Stack
+resource "turbot_policy_setting" "aws_backup_stack" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-backup#/policy/types/backupStack"
+  #value    = "Check: Configured"
+  value    = "Enforce: Configured"
+}
+
+# AWS > IAM > Stack > Source
+resource "turbot_policy_setting" "aws_iam_iam_stack" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-iam#/policy/types/iamStack"
+  # value    = "Check: Configured"
+  value    = "Enforce: Configured"
+}
+```
+
+Then re-apply the changes:
+
+```sh
+terraform plan
+terraform apply
+```
+
+## Disable Backups
+- To stop backups from happening while preserving the existing backups, remove this resource definition from the `AWS > Backup > Stack > Source` policy setting. With no resource
+```hcl
+    resource "aws_backup_selection" "ebs_resource_assignment" {
+      iam_role_arn = "arn:aws:iam::{{ $.account.id }}:role/turbot/core/guardrails_backup_service_role"
+      name         = "guardrails-ebs-resource-assignment"
+      plan_id      = aws_backup_plan.guardrails_ebs_backups.id
+      resources = ["arn:aws:ec2:*:*:volume/*"]
+    }
+```
+- Commit the updated policy setting via Terraform then let the `AWS > Backup > Stack` make the changes. 
+
+
+## Decommission the Backup Vault
+To completely remove a Backup Vault and all backups, execute the following steps:
```

Comment:
Under this ` Decommission the Backup Vault` section, maybe we could have 3 H3 sections:
- Delete EBS Volume Backups
- Delete Backup Vault
- Delete IAM Role

It would help break up this section, as it feels a bit large, especially with the code samples, which are helpful to include